libs/designsystem/src/lib/scss/_global-styles.scss

```diff
@@ -2,6 +2,7 @@
 @import './base/ionic';
 @import './base/typography';
 @import './base/line-clamp';
+@import './base/lists';
```

Comment:
I think we should rename the partial to singular "list"

Reply:
👉🏻 `@import './base/list';` - and of course the file needs to be renamed as well